src/rules/vue-recommended/elementAttributeOrder.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-cond-assign */
+import type { SFCTemplateBlock } from '@vue/compiler-sfc'
+import { BG_ERR, BG_RESET, BG_WARN, TEXT_INFO, TEXT_RESET, TEXT_WARN } from '../asceeCodes'
+import { getUniqueFilenameCount } from '../../helpers'
+
+interface ElementAttributeOrder { filename: string, message: string }
+
+const elementAttributeOrderFiles: ElementAttributeOrder[] = []
+
+const ATTRIBUTE_ORDER = [
+  'is',
+  'v-for',
+  'v-if',
+  'v-else-if',
+  'v-else',
+  'v-show',
+  'v-cloak',
+  'v-pre',
+  'v-once',
+  'id',
+  'ref',
+  'key',
+  'v-model',
+  'v-on',
+  'v-html',
+  'v-text',
+]
+
+const checkElementAttributeOrder = (template: SFCTemplateBlock | null, filePath: string) => {
+  if (!template)
+    return
+
+  // Remove the <template> tags to avoid checking them
+  const innerTemplate = template.content.replace(/<\/?template>/g, '')
```

Comment:
Can you have magic-regexp here for any of the regexes?

Reply:
I tried, but the result from `convert` was not working as expected. So I think in a next iteration of the rule, I'd have more knowledge with `magic-regexp`